amazon-chime-sdk-js/src/transceivercontroller/DefaultTransceiverController.ts

```diff
@@ -574,6 +585,251 @@ export default class DefaultTransceiverController
     this.addRedundantAudioRecoveryMetricsObserver(this.meetingSessionContext?.statsCollector);
   }
 
+  protected setupInsertableStreamWorker(): void {
+    // @ts-ignore
+    const supportsRTCScriptTransform = !!window.RTCRtpScriptTransform;
+    // @ts-ignore
+    const supportsInsertableStreams = !!RTCRtpSender.prototype.createEncodedStreams;
+
+    if (supportsRTCScriptTransform) {
+      // This is the prefered approach according to
+      // https://github.com/w3c/webrtc-encoded-transform/blob/main/explainer.md.
+      this.logger.info(
+        '[InsertableStream] Supports encoded insertable streams using RTCRtpScriptTransform'
+      );
+    } else if (supportsInsertableStreams) {
+      this.logger.info(
+        '[InsertableStream] Supports encoded insertable streams using TransformStream'
+      );
+    } else {
+      this.disableAudioRedundancy();
```

Comment:
We probably want to disable AudioRedundancy when enableInsertableStream is true. This call would probably live outside of this setupInsertaableStreamWorker function